### PR TITLE
Release v1.21.3 : RDM submission download / markdown fixes

### PR DESCRIPTION
**Pull request by jmgasper**
* RDM submission download
* Markdown rendering fixes
* Uni-nav loading indicator fix

### CODE REVIEW

File: .eslintrc [hunk 1]
```diff
@@ -3,9 +3,13 @@
   "rules": {
     "jsx-a11y/anchor-is-valid": false,
     "import/no-cycle": [2, { "maxDepth": 1 }],
-    "react/forbid-prop-types": false
+    "react/forbid-prop-types": false,
+    "react/no-unknown-property": false,
+    "react/jsx-no-bind": false,
+    "react/destructuring-assignment": false,
+    "react/no-array-index-key": false
```
Comment (luizrrodrigues) [inline .eslintrc:10]: Disabling lint rules instead fixing the issues is not best practice.

1. Are easy to fix and affect just some files:
```
"react/jsx-no-bind": false,
"react/destructuring-assignment": false,
"react/no-array-index-key": false
```
2. This affect a lot of files and need some work to update all.
```
"react/no-unknown-property": false,
```
-----

I think these issue coming with bump from React version.

File: src/shared/components/Leaderboard/PodiumSpot/index.jsx [hunk 2]
```diff
@@ -1,3 +1,4 @@
+/* eslint-disable react/prop-types */
```
Comment (luizrrodrigues) [inline src/shared/components/Leaderboard/PodiumSpot/index.jsx:1]: ESLint disabled too, ref. above.

**Review by luizrrodrigues — CHANGES_REQUESTED** [review]

File: .eslintrc (hunk 1 above)
Comment (luizrrodrigues) [reply .eslintrc:10]: cc @jmgasper

Comment (jmgasper) [reply .eslintrc:10]: @luizrrodrigues - Ok, can we approve this request, and then I'll get this fixed for the next build please?  I don't have time to fix this for our scheduled deploy today.

Comment (luizrrodrigues) [reply .eslintrc:10]: Hey @jmgasper sure, let move forward with that and track these issues.

Please share the Jira ticket to track this.

**Review by luizrrodrigues — APPROVED** [review]

File: .eslintrc (hunk 1 above)
Comment (jmgasper) [reply .eslintrc:10]: https://topcoder.atlassian.net/browse/PROD-4153

**Pull request merged**